portal/meson.build

```diff
@@ -30,6 +34,7 @@ portal_dir = datadir / 'xdg-desktop-portal' / 'portals'
 conf = configuration_data()
 conf.set('libexecdir', prefix / libexecdir)
 conf.set('dbus_name', dbus_name)
+conf.set('service_name', service_name)
```

Comment:
why not use meson.project_name?

Reply:
I set `service_name = meson.project_name()`.